README.md

```diff
@@ -30,9 +30,9 @@ Changes to the site can be tested locally before deploying to the web. To get se
   - this will trigger the heroku deploy to production. 
  
 #### Manual Releases
-Currently, we deploy to Heroku. Our application is small enough that it is free to host on Heroku, and we dont mind the server having to wake up if nobody has accessed the site in a while.
+Currently, we deploy to Heroku. Our application is small enough that it is free to host on Heroku, and we can wake up the server if the site has not been accessed in a while.
```

Comment:
I think we're actually using a paid tier now. I'll double check and we can probably delete this line if so.